src/Components/AllRides/AllRides.js

```diff
@@ -3,22 +3,28 @@ import './AllRides.css'
 import Ride from '../Ride/Ride'
 
 
-const AllRides = ({ allRides }) => {
+function AllRides({ allRides }) {
 
     const displayAllRides = allRides.map(ride => {
         return (
-            <Ride key={Date.now()} ride={ride} />
+            <Ride key={ride.date} ride={ride} />
```

Comment:
I think keys are supposed to be unique, I wonder if you'll run into any trouble if you have rides with the same date booked?